UT failure for EtcdMetadataReport

```
[INFO] Running org.apache.dubbo.metadata.store.etcd.EtcdMetadataReportTest
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
        â„¹ï¸Ž Checking the system...
        âœ” Docker version should be at least 1.6.0
        âœ” Docker environment should have more than 2GB free disk space
```

```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.apache.dubbo.metadata.store.etcd.EtcdMetadataReportTest
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 10.492 s <<< FAILURE! - in org.apache.dubbo.metadata.store.etcd.EtcdMetadataReportTest
[ERROR] testStoreProvider  Time elapsed: 0.009 s  <<< ERROR!
java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration
	at org.apache.dubbo.metadata.store.etcd.EtcdMetadataReportTest.<init>(EtcdMetadataReportTest.java:54)

[ERROR] testStoreConsumer  Time elapsed: 0 s  <<< ERROR!
java.lang.IllegalStateException: Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration
	at org.apache.dubbo.metadata.store.etcd.EtcdMetadataReportTest.<init>(EtcdMetadataReportTest.java:54)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   EtcdMetadataReportTest.<init>:54 » IllegalState Previous attempts to find a Do...
[ERROR]   EtcdMetadataReportTest.<init>:54 » IllegalState Could not find a valid Docker ...
[INFO] 
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0
```
Appropriate docker environment is required for UT now.

--- a/dubbo-metadata-report/dubbo-metadata-report-etcd/src/test/resources/testcontainers.properties
+++ b/dubbo-metadata-report/dubbo-metadata-report-etcd/src/test/resources/testcontainers.properties
@@ -1,0 +1,1 @@
+checks.disable=true

